### DISCUSSION TITLE
blog/posts/230128-gatsby-6-toc/

### DISCUSSION THREAD

**Discussion by giscus** [ricale/blog#53] [Giscus]
# blog/posts/230128-gatsby-6-toc/

이전 글들에서는 Gatsby 기본 기능을 구현을 위한 튜토리얼을 소개했고 태그 기능을 구현했고, MDX 글 안에 인라인 이미지와 테이블을 삽입했으며, 시리즈 기능도 만들어보았고, 임시글 기능도 구현했고. Google Analytics 도 적용해보았다.…

https://ricale.kr/blog/posts/230128-gatsby-6-toc/

**Comment by areumsheep** [comment]
게츠비 블로그 시리즈 너무 잘 읽고 있어요! 
저도 블로그 개발하고 있는데 큰 도움이 됩니다!

감사해요 :)

**Reply by ricale** [reply]
도움이 되었다니 다행이네요 😄

**Comment by yanggwi** [comment]
https://ricale.kr/blog/posts/230128-gatsby-6-toc/#23-gatsby-remark-table-of-contents-%ED%94%8C%EB%9F%AC%EA%B7%B8%EC%9D%B8

여기 중간에 "gatsby-node.ts 에 설정을 추가해주자." gatsby-node.ts 가 아니고 gatsby-config.ts 인거 같네~

**Reply by ricale** [reply]
앗 그렇네요. 감사합니다, 수정해둬야겠네요 😅
